tests/sphere_test.cpp

```diff
@@ -0,0 +1,104 @@
+#include "obj/objects.h"
+#include "gtest/gtest.h"
+
+#include <random>
+
+using namespace core;
+using objects::Sphere;
+
+const float kSkipFreq = 1000;
+
+class RandomFloatGenerator {
+public:
+  RandomFloatGenerator(float mean, float standardDeviation)
+      : engine_(static_cast<std::mt19937_64::result_type>(std::time(nullptr))),
```

Comment:
Does this initialize engine_ with a constant seed? if yes, why? if no, also why? and then fix it if not.

Reply:
sed is not constant., because everytime time is different, so no :P. I deletad ranomness in this test :-)